secure-app-model/keyvault/cpvsample/src/main/java/com/microsoft/store/samples/secureappmodel/cpvsample/security/AccessTokenProvider.java

```diff
@@ -161,7 +161,8 @@ public AuthenticationResult getAccessTokenBySecureRefreshToken(String tenantId,
         IVaultProvider vault = new KeyVaultProvider(
```

Comment:
We shouldn't need this if we are using DefaultAzureCredential

Reply:
we shouldn't need a reference to adal4j anymore. azure-identity uses msal underneath.

Reply:
The adal4j not only used to key vault but also used to Azure manage SDK and track 1 graph client. So I don't think it can be removed.